gce_batch_job.json

```diff
@@ -20,8 +20,8 @@
                 ],
                 "computeResource": {
                     "cpuMilli": 48000,
-                    "memoryMib": 183105,
-                    "boot_disk_mib": 190735
+                    "memoryMib": 190000,
+                    "boot_disk_mib": 300000
```

Comment:
I will be progressively bringing down these requirements while monitoring the ongoing jobs.